cypress/fixtures/data-sets.json

```diff
@@ -0,0 +1,134 @@
+{
+    "dataSets": [
+        {
+            "id": "lyLU2wR22tC",
+            "displayName": "ART monthly summary",
+            "periodType": { "name": "Monthly" }
```

Comment:
This fixture will need to be updated once https://jira.dhis2.org/browse/DHIS2-11198 is resolved.